db/migrate/20220831160429_create_articles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateArticles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :body
+
+      t.timestamps
```

Comment:
When I try to populate the database with data to test your views I get an error, specifically with creating posts.

![Screenshot from 2022-08-31 23-22-30](https://user-images.githubusercontent.com/38088818/187788291-180a8c4c-64a6-43cb-b210-e5efe918bd8c.png)

This happens when I try to creating a post by running:
`first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post')`

after creating users with;
```
   first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
   second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
```

and comments with:
`Comment.create(post: first_post, author: second_user, text: 'Hi Tom!' )`

If you are using a different format to insert data into your database, kindly create sample data in your `db/seed.rb` file which can be run to populate your database.

Please make sure that:

- There's at least one user
- At least 4 posts written by one of the users you created
- At least 6 posts comments for one of the posts you created